### ISSUE TITLE
ONYX-399: Crash when trying to list comment for an interview

### ISSUE THREAD

**Issue by ymarcon** [obiba/onyx#431]
Jira issue originally created by user mboulang:

I get the following crash when I try to list the interview comments (click on comments in the interview page) for participant with id "1111111".  This is was tested in the simulation room with Ontario's deployment configuration.  I don't have the specific steps to recreate this, however you will find a dump of the database attached to this issue.

2009-03-19 16:31:43,218 - ERROR - apache.wicket.RequestCycle - Exception in rendering component: [Component id = rowValue]
org.apache.wicket.WicketRuntimeException: Exception in rendering component: [Component id = rowValue]
	at org.apache.wicket.Component.renderComponent(Component.java:2519)
	at org.apache.wicket.markup.html.WebComponent.onRender(WebComponent.java:60)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1401)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.markup.repeater.AbstractRepeater.renderChild(AbstractRepeater.java:119)
	at org.apache.wicket.markup.repeater.AbstractRepeater.onRender(AbstractRepeater.java:100)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:639)
	at org.apache.wicket.markup.html.panel.Panel.onComponentTagBody(Panel.java:112)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1401)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.markup.repeater.AbstractRepeater.renderChild(AbstractRepeater.java:119)
	at org.apache.wicket.markup.repeater.AbstractRepeater.onRender(AbstractRepeater.java:100)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1401)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:639)
	at org.apache.wicket.markup.html.panel.Panel.onComponentTagBody(Panel.java:112)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
	at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:639)
	at org.apache.wicket.markup.html.panel.Panel.onComponentTagBody(Panel.java:112)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
	at org.apache.wicket.Component.render(Component.java:2317)
	at org.apache.wicket.Component.renderComponent(Component.java:2420)
	at org.apache.wicket.ajax.AjaxRequestTarget.respondComponent(AjaxRequestTarget.java:785)
	at org.apache.wicket.ajax.AjaxRequestTarget.respondComponents(AjaxRequestTarget.java:662)
	at org.apache.wicket.ajax.AjaxRequestTarget.respond(AjaxRequestTarget.java:577)
	at org.apache.wicket.request.AbstractRequestCycleProcessor.respond(AbstractRequestCycleProcessor.java:104)
	at org.apache.wicket.RequestCycle.processEventsAndRespond(RequestCycle.java:1181)
	at org.apache.wicket.RequestCycle.step(RequestCycle.java:1252)
	at org.apache.wicket.RequestCycle.steps(RequestCycle.java:1353)
	at org.apache.wicket.RequestCycle.request(RequestCycle.java:493)
	at org.apache.wicket.protocol.http.WicketFilter.doGet(WicketFilter.java:355)
	at org.apache.wicket.protocol.http.WicketFilter.doFilter(WicketFilter.java:200)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.springframework.orm.hibernate3.support.OpenSessionInViewFilter.doFilterInternal(OpenSessionInViewFilter.java:198)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.util.MissingResourceException: Unable to find resource: Action.Reason.ParticipantWithdrewConsent for component: addCommentsModal:content:previousComments:comment-list [class=org.obiba.onyx.webapp.participant.panel.CommentsModalPanel$CommentsDataView]
	at org.apache.wicket.Localizer.getString(Localizer.java:266)
	at org.apache.wicket.model.StringResourceModel.getString(StringResourceModel.java:478)
	at org.apache.wicket.model.StringResourceModel.load(StringResourceModel.java:621)
	at org.apache.wicket.model.LoadableDetachableModel.getObject(LoadableDetachableModel.java:114)
	at org.obiba.wicket.markup.html.panel.KeyValueDataPanel$3.getObject(KeyValueDataPanel.java:260)
	at org.apache.wicket.Component.getModelObject(Component.java:1558)
	at org.apache.wicket.Component.getModelObjectAsString(Component.java:1580)
	at org.apache.wicket.markup.html.basic.Label.onComponentTagBody(Label.java:111)
	at org.apache.wicket.Component.renderComponent(Component.java:2480)
	... 74 more
